build/Microsoft.DotNet.Cli.Test.targets

```diff
@@ -71,34 +72,31 @@
                             SetupTestProjectData;"
           Inputs="%(TestProjects.BuildInputs)"
           Outputs="%(TestProjects.BuildOutput)">
-
-    <DotNetBuildPj  Configuration="$(Configuration)"
-                    Framework="%(TestProjects.Framework)"
-                    ToolPath="$(Stage0PjDirectory)"
-                    WorkingDirectory="%(TestProjects.RelativeDir)" />
+    <DotNetBuild  Configuration="$(Configuration)"
+                  Framework="%(TestProjects.Framework)"
+                  ToolPath="$(Stage0Directory)"
+                  WorkingDirectory="%(TestProjects.RelativeDir)" 
```

Comment:
We might not. I can try taking it out (in the next PR though) if this turns green I want to get this checked in.